pkg/apis/ark/v1/zz_generated.deepcopy.go

```diff
@@ -1,7 +1,7 @@
 // +build !ignore_autogenerated
 
 /*
-Copyright 2018 the Heptio Ark contributors.
+Copyright 2019 the Heptio Ark contributors.
```

Comment:
I think this file might be out of date - https://github.com/heptio/ark/commit/e38e2fc5a1e2dcd94ce135d925c5172b7b7b6b8b removed the year from the copyright lines.

Reply:
Oh, nevermind - this is on the `plugins` branch, not `master`. My mistake.